samples/notebooks/KubeFlow Pipeline Using TFX OSS Components.ipynb

```diff
@@ -32,9 +32,7 @@
    "source": [
     "# Install Pipeline SDK\n",
     "!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.1/kfp.tar.gz --upgrade\n",
-    "    \n",
```

Comment:
I thought we just added this. Why do we delete the kubernetes installation?

Reply:
We have that in setup.py now.